src/osd/PG.cc

```diff
@@ -1587,8 +1581,14 @@ void PG::on_role_change() {
 }
 
 void PG::on_new_interval() {
-  dout(20) << __func__ << " scrub_queued was " << scrub_queued << " flags: " << m_planned_scrub << dendl;
-  scrub_queued = false;
+  // log some scrub data before we react to the interval
+  if (m_scrubber) {
```

Comment:
This should probably be an assert, shouldn't be possible to invoke on_new_interval() on a destructed PG.

Reply:
fixed